cfgov/v1/models/base.py

```diff
@@ -247,6 +247,33 @@ def get_streamfield_content(self, section, blockType, value):
                 return self.remove_html_tags(item.value[value].source)
         return
 
+    def get_meta_description(self):
+        """Deliver a meta_description, following a preference order."""
+        preference_order = [
+            'header_hero_body',
+            'preview_description',
+            'header_text_intro',
+            'content_text_intro',
+            'header_item_intro',
+        ]
+        candidates = {item: None for item in preference_order}
```

Comment:
There's no particular need to set all of the keys, is there?

```suggestion
        candidates = {}
```